pkg/controllermanager/controller/project/project_control_reconcile.go

```diff
@@ -216,3 +247,48 @@ func (c *defaultControl) reconcileNamespaceForProject(project *gardencorev1beta1
 
 	return namespace, nil
 }
+
+// CustomProjectRole is a constant for a label value.
+const CustomProjectRole = "custom-project-role"
```

Comment:
Can we put this const into `pkg/apis/core/{v1alpha1,v1beta1}/constants/types_constants.go` or is there a reason, why it is placed here?